fixed vm machine setting and added iptables to apt-get task

richierampage opened the pull request:
Left in change to vagrantfile, removed variable settings and added iptables to apt-get. 

Still need to resolve mkdocs to get full provision but this fixes vmware vm not booting.  

:-)

philipnorton42 commented:
Thanks for this @richierampage. I've also removed the mkdocs install as it was only there to help build the documentation, which we have moved to a different repo anyway :)

richierampage commented:
cool, I'll give it a pull and try. Should complete vmware build now, will
confirm.

On 29 January 2015 at 10:25, Philip Norton <notifications@github.com> wrote:

> Thanks for this @richierampage <https://github.com/richierampage>. I've
> also removed the mkdocs install as it was only there to help build the
> documentation, which we have moved to a different repo anyway :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hashbangcode/vlad/pull/136#issuecomment-72002358>.
>



-- 
Office:  01732 874054
Mobile: 07734 733541
Skype: richard.moger
Twitter: richardmoger <http://twitter.com/richardmoger>
